pages/jobs/[slug].tsx

```diff
@@ -83,7 +83,10 @@ export async function getStaticProps({
 }) {
     try {
         const job = await getJobPost(params.slug);
-        const content = (await getJobPage(preview)) ?? [];
+        const content = await getJobPage(preview);
+        if (!job) {
+            return { notFound: true };
+        }
```

Comment:
The try catch won't report an exception here if the job is returned as undefined. Hence I've added an if statement to redirect to the 404 page - however, we could instead have `getJobPost` to throw an error when a page is missing if this is neater

Reply:
404 seems like the appropriate response here. you should already be throwing errors related to people HR from the snippet below (inside peopleHRJobPostings). One werid bug/usecase I've just noticed from checking this logic is how the server would handle no job postings, it looks like we'd just cache the old jobpostings if we wanted to intentionally remove them all from people HR, I might be wrong on this though as I haven't tested it 

```
            const peopleHRJobPostings = await fetchPeopleHRFeed().catch(
                function (error) {
                    console.error('Error fetching data from PeopleHR: ', error);
                    return null;
                },
            );
```

Reply:
I will add a ticket for this as part of adding Sentry logging to the site 👍

Reply:
The cache should store an empty jobs feed, returning zero jobs as expected